setup.py

```diff
@@ -29,5 +29,6 @@
         'pyyaml',
         'jsonschema',
         'simplejson',
+        'functools32',
```

Comment:
I'm curious what is missing without this. From my understanding, only sockets with SSL support is required. I'm slightly hesitant to drop support for python2.6.